channels/serializers/channels.py

```diff
@@ -126,6 +127,21 @@ def validate_allowed_post_types(self, value):
 
         return {"allowed_post_types": value}
 
+    def get_about(self, channel):
+        """get the value for the about field"""
+        channel_obj = channel._self_channel  # pylint: disable=protected-access
```

Comment:
We want to have this kind of in `channels.proxies.ChannelProxy` instead of the serializer so that when we change the object type being serialized from that to an actual `Channel` object the serializer doesn't need to be updated.

So I'd suggestion putting an `@property` in `ChannelProxy` for `about` and just making this a `JSONField`. That'll give you full JSON validation too so you'll be able to drop both your `get_about` and `validate_about` implementations in one fell swoop too.

Reply:
I added `"about"` to the list of fields to look up on `self._self_channel` in the `__getattr__` method on `ChannelProxy`, does that accomplish the same thing?

Reply:
It seems to work ok in the API when I test it